src/components/OrganizationsTable.tsx

```diff
@@ -1,45 +1,58 @@
 import React from "react";
 import gql from "graphql-tag";
-import { Table } from "antd";
+import { Table } from 'antd';
```

Comment:
I'd try to keep the `"` convention if you can

Reply:
Oh, of course! Extra reckless of me since I don't really know the difference between single and double quotes in javascript (is one literal and the other allows variable substitutions, maybe that's ruby...)